components/prolayout/menu-item.component.ts

```diff
@@ -22,14 +16,18 @@ export class VtsProlayoutMenuItemComponent implements OnInit {
     isSelected: false
   };
   defaultIcon: string = 'AddDoutone:vts';
+  currentUrl: string = "";
 
-  constructor(private router: Router) {}
+  constructor(private router: Router, private cdf: ChangeDetectorRef) {
+    this.currentUrl = this.router.url;
+    this.router.events.subscribe(evt => {
```

Comment:
Cần unsubscribe khi destroy. Em bổ sung Provider VtsDestroyService ở Component và takeUntil(this.vtsDestroyService) nhé